src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -326,37 +342,43 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
             case (Right(rawlsGroup:RawlsGroup), level) => RawlsGroup.toRef(rawlsGroup)
           }.toSet
 
-          val groupsByLevel: Map[WorkspaceAccessLevel, Map[Either[RawlsUser, RawlsGroup], WorkspaceAccessLevel]] = updateMap.groupBy({ case (key, value) => value })
-          //go through the access level groups on the workspace and update them
-          workspaceContext.workspace.accessLevels.foreach { case (level, groupRef) =>
-            withRawlsGroup(groupRef, txn) { group =>
-              //remove existing records for users and groups in the acl update list
-              val users = group.users.filter( userRef => !allTheRefs.contains(userRef) )
-              val groups = group.subGroups.filter( groupRef => !allTheRefs.contains(groupRef) )
-
-              //generate the list of new references
-              val newusers = groupsByLevel.getOrElse(level, Map.empty).keys.collect({ case Left(ru) => RawlsUser.toRef(ru) })
-              val newgroups = groupsByLevel.getOrElse(level, Map.empty).keys.collect({ case Right(rg) => RawlsGroup.toRef(rg) })
+          if (allTheRefs.contains(UserService.allUsersGroupRef)) {
```

Comment:
this is where acl changes are prevented

the rest of the changes in this method are just indenting